python/tk_multi_workfiles/file_form_base.py

```diff
@@ -282,7 +282,7 @@ def _fileitem_from_path(self, path, work_area):
         :param work_area:   A WorkArea instance representing the work area the file is in
         :returns:           A FileItem representing the specified path in the specified work area
         """
-        if not path or not work_area or not work_area.work_template:
+        if not path or not work_area:
```

Comment:
A couple lines under this there's a second test for Falsyness, so I removed this one that was redundant.